spec/models/generic_file_spec.rb

```diff
@@ -634,4 +634,104 @@ def paranoid_edit_permissions
       it { is_expected.not_to be_public }
     end
   end
+
+  describe "find_by_date_created" do
+    subject { GenericFile.find_by_date_created start_date, end_date }
```

Comment:
Please add parens around `start_date` and `end_date` for consistency with the other calls to `#find_by_date_created`